src/events/processor/find-features/index.js

```diff
@@ -139,9 +139,10 @@ const generateFeatures = ({ locations, report, options, sourceRatings }) => {
       }
     }
 
-    // Store coordinates on location
+    // Store coordinates and area on location
     if (feature.geometry) {
       location.coordinates = turf.center(feature).geometry.coordinates;
+      location.area = turf.area(feature.geometry);
```

Comment:
Think we ought to include `area` in the final output? Any reason it would be useful (besides, of course, calculating `populationDensity`, which we're already doing).

Reply:
We can always add it if someone asks.